src/vm_cpu.rs

```diff
@@ -29,6 +29,7 @@ use crate::vm::MmioOperation;
 use crate::vm_id::VmId;
 use crate::vm_pages::{ActiveVmPages, VmPages};
 use crate::{println, CONSOLE_DRIVER};
+use PmuCounterState::*;
```

Comment:
This puts very generic names like `Configured`, `Stopped`, etc. in the top-level namespace in this module. Could we leave this scoped to methods that actually need it?

Reply:
Not resolved.